ingestion/src/metadata/ingestion/source/database/athena.py

```diff
@@ -34,12 +36,12 @@
 logger = ingestion_logger()
 
 
-def _get_column_type(self, type_):
+def _get_column_type(self, type_):  # pylint: disable=too-many-branches
```

Comment:
I think we should not disable that. To me it means we need to maybe change our deep if/else to a look dict or soemthing similar -- example below.

```python
def bad(a):
    if a == 1: return "A"
    if a == 2: return "B"
    if a == 3: return "D"
    if a == 4: return "G"
    if a == 5: return "M"
    if a == 6: return "X"

    return "Z"

def better(a):
    return _LOOKUP.get(a, "Z")

_LOOKUP = {
    1: "A",
    2: "B",
    3: "D",
    4: "G",
    5: "M",
    6: "X",
}
```